caffe2/core/export_c10_op_to_caffe2.h

```diff
@@ -1,13 +1,16 @@
 #pragma once
 
+#include <c10/macros/Macros.h>
+#include <c10/util/Registry.h>
+#include "caffe2/core/operator.h"
```

Comment:
Some downstream deps (not in xplat, apparently?) seem to rely on this being in scope, so I whisked it out of the ifdef.